README.md

```diff
@@ -68,53 +68,40 @@ The idea is to install "Legacy Version" of WhatsApp on you device via adb and us
 
-### Prerequisites
+### Prerequisites and preparation
 
 * USB Debugging must be enabled on the target device. Settings -> Developer Options -> (Debugging) USB debugging  
-     If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number multiple times until you're finally declared a developer. 
+  * If you cannot find Developer Options then please go to: Settings -> About phone/device and tap the Build number multiple times until you're finally declared a developer. 
 * Android device with Android 11 or higher with activated 'ADB over network' setting. Settings -> Developer Options -> ADB over network
 * OR an Android device with Android 4 to 10 (i.e. Ice Cream Sandwich, Jelly Bean, KitKat, Lollipop, Marshmallow, Nougat, Oreo, Pie, Q) AND a PC with USB functionality and installed [Android Debug Bridge for PC](https://developer.android.com/studio/releases/platform-tools)
   * Run the following commands in a terminal opened in the 'platform-tools' folder. Authorize on your phone when prompted:
-  * `adb devices`
-  * `adb tcpip 5555`
+   * `adb devices`
+   * `adb tcpip 5555`
 * Termux : Install from [Play Store](https://play.google.com/store/apps/details?id=com.termux) or [F-Droid](https://f-droid.org/packages/com.termux)
 
-Run the following commands in the Termux app:
-* Python 3.x : `pkg install python -y`
-* Proot : To make Java work `pkg install proot -y && proot login`
-* [Java](https://github.com/Hax4us/java/) : `pkg update && pkg install wget -y && wget https://raw.githubusercontent.com/Hax4us/java/master/installjava && sh installjava`
-* [ADB (Android Debug Bridge)](https://github.com/MasterDevX/Termux-ADB) Drivers : `pkg update && pkg install wget && wget https://github.com/MasterDevX/Termux-ADB/raw/master/InstallTools.sh && bash InstallTools.sh`
-* Git : `pkg install git`
-
+![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
+### Installation and Usage 
+Note: run all the commands in Termux
 
-![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
+1. Gathering prerequsites
+`pkg update && pkg upgrade`
+`pkg install python git`
 
-### Installation (Not working?)
+2. Clone the repo
+`git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git`
 
-0. Connect adb to your phone. Run the following command in the Termux window: `adb connect localhost`
+3. Get Termux requirements
+`python tr.py`
 
-1. Clone the repo
-```
-git clone https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor.git && cd WhatsApp-Key-Databse-Extractor
-```
-2. Get python requirements
-```python
-pip install -r requirements.txt
-```
-3. Get Termux dependencies (skip if already installed proot, adb, java etc.)
-```python
-python helpers/termux_dependencies.py
-```
```

Comment:
Let them be like this, looks good that way.

Reply:
Thats an outdated version, it now looks like this:
![image](https://user-images.githubusercontent.com/31773450/108526823-e0ca0780-72d1-11eb-9421-10b4ffd748ec.png)

Reply:
No I meant change the content to be updated but don't change styling.

Reply:
For me it looks exactly the same, weird

Reply:
Okay, changed it back